src/playwrightTest.ts

```diff
@@ -66,6 +66,30 @@ export class PlaywrightTest {
   }
 
   async getPlaywrightInfo(workspaceFolder: string, configFilePath: string): Promise<{ version: number, cli: string } | null> {
+
+    if (isPnPWorkspace(workspaceFolder)) {
```

Comment:
would be also good if we can move it out to a different file, add some more handlers, hooks etc. so playwrightTest.ts is not so much involved.

Reply:
So you would like to move all pnp-related utility functions into your own file?

What do you mean by "why node -e does not work"?